src/Compilers/CSharp/Test/Emit/PDB/PDBAsyncTests.cs

```diff
@@ -159,7 +159,7 @@ .maxstack  3
    -IL_00e5:  ldsfld     ""System.Threading.AutoResetEvent Driver.CompletedSignal""
     IL_00ea:  callvirt   ""bool System.Threading.EventWaitHandle.Set()""
     IL_00ef:  pop
-   ~IL_00f0:  leave.s    IL_010c
+    IL_00f0:  leave.s    IL_010c
```

Comment:
Yes. This sequence point was added by my original fix (https://github.com/dotnet/roslyn/pull/16803/files, which only handled conditional in async method). It it not essential because the only way to get there is through the code right above (there are no conditional jumps to the `leave.s`).